internal/executors/condition/condition.go

```diff
@@ -0,0 +1,58 @@
+package condition
+
+import (
+	"errors"
+	"reflect"
+	"soarca/logger"
+	"soarca/models/cacao"
+	"soarca/models/execution"
+	"soarca/utils/stix"
+)
+
+var component = reflect.TypeOf(Executor{}).PkgPath()
+var log *logger.Log
+
+func init() {
+	log = logger.Logger(component, logger.Info, "", logger.Json)
+}
+
+func New(stix stix.IStix) *Executor {
+	return &Executor{stix: stix}
+}
+
+type IExecuter interface {
```

Comment:
Just for clarity - perhaps it would be clearer to rename this interface to something like IIfConditionExecuter?

Reply:
I tried to keep it short because it's already in the package condition